packages/types/src/coreExtensions/settings.ts

```diff
@@ -38,7 +38,7 @@ export type Settings = {
     label: string,
     element: React.FunctionComponent,
     color?: string | null,
-    pos?: number,
+    pos?: number | ((section: unknown) => number),
```

Comment:
Is there a specific reason these are `unknown` instead of `SettingsSection[]`?

Reply:
oops my bad, I was moving things around and forgot to add it back, will fix that lmao